docs/database-classes.md

```diff
@@ -350,7 +350,7 @@ transactor.Transact(db =>
 
 * All database classes have to be declared as `public abstract`.
 * Database classes may not inherit non-database classes.
-* Database classes shall have a public parameterless constructor.
+* Database classes shall have either no declared constructor, or a public parameterless constructor.
```

Comment:
Does this sentence mean that only public parameterless constructors are supported? Then for me, it would be easier to this way.

Reply:
Non-public constructors are supported as long as there is a public parameter-less constructor.

Reply:
In c# terms, it means we require a public parameterless constructor. Which might be a user- or automatically-defined. For me, it's hard to follow this either/or logic. Which still implies, that a reader should know about automatically-defined constructors.

Reply:
@erikvk could you suggest a change here? Please.